content/concepts/networks-and-servers/the-clio-server.md

```diff
@@ -36,6 +36,13 @@ A Clio server stores validated ledger data such as transaction metadata, account
 
 When a Clio server receives an API request, it looks up data from these data stores. For requests that require data from the P2P network, the Clio server forwards the request to a P2P server, and then passes the response back to the client.
 
+Clio will **always** forward to `rippled` if any of the following is true:
+
+- `ledger_index` is set to `current` or `closed`.
+- `api_version` is set to `1` - Clio only supports API v2.
```

Comment:
```suggestion
- `api_version` is set to `1` - Clio currently only supports API v2 and therefore defaults to `api_version: 2`. **Caution:** This causes potential incompatibility with the `rippled` APIs. See the [comparison of the differences between API versions](https://github.com/xrplf/rippled/blob/develop/API-CHANGELOG.md).
```

Technically, API v2 has not been defined yet, and it will likely include other changes such as:
- https://github.com/XRPLF/rippled/issues/3484
- https://github.com/XRPLF/clio/issues/722


So what we have in Clio is not the final version of API v2, but just the "beta" (incomplete) API v2 - perhaps more accurately described as "API v1.5".

Reply:
Update: Clio will implement API v1 to ensure no breaking change for existing clients

Reply:
Yea, we can actually drop this entire PR now.